### ISSUE TITLE
App crashes and disappears

### ISSUE THREAD

**Issue by Tony20221** [QuiteRSS/quiterss#1558]
This app crashes for no apparent reason and it seems it happens when I am moving around in the episodes list. I can handle the crashing once in a while but today after I added a few feeds it crashed and I lost all the new feeds. That was pretty annoying. Luckly I use a clipboard manager.

The app needs to add the new feed to the configuration file as soon as it's added by the user. Don't wait till the app closes to update the file. This way if the app crashes, nothing is lost.

My advice also to the users is that once you added the feeds, close the app right away so that it saves them quickly then relaunch the app.

**Comment by TheOne320** [comment]
I am having the same problem. It usually happens when I click through many feeds quickly in succession. The app crashes and it forgets all "read" feeds in the time I went through them. 
I already reported the problem >4 years ago and it was never fixed.
https://github.com/QuiteRSS/quiterss/issues/1091

**Comment by Rex000** [comment]
Same issue here. Really sick of this app crashing so often.

**Comment by jziggy101** [comment]
Same issues as reported by others.  I have it running in the background & when I check on it a few hours later its gone & not running.  As I clean posts in the feeds & delete them the app will suddenly shutdown. When restarted any lines deleted or marked 'read' are now back again & read items are listed as non read.   
This is a fine app and I really like the flexibility but the bugs are making it hard to enjoy.

**Comment by Rex000** [comment]
Yep it's been happening a lot lately. Looks like they have abandoned the app which is a shame because there is no such alternative to it that's as good.

**Comment by vitusson** [comment]
No such problems at all here on Debian stable

**Comment by Rex000** [comment]
Maybe you don't have a lot of feeds. Either that or the app just lost it's compatibility with Windows.

**Comment by jziggy101** [comment]
I am running under Windows.  I have 28 feeds I pull from.  I have gone in and reduced the times during which the updates to the feeds happen hoping that would help but it did not.

**Comment by jziggy101** [comment]
> Yep it's been happening a lot lately. Looks like they have abandoned the app which is a shame because there is no such alternative to it that's as good.

I agree there is NO other app that is as flexible.  I wish they would still be supporting it.

**Comment by Tony20221** [comment]
It crashes a lot less if you don't click around too fast. For example, let the preview pane finish rendering before you click somewhere else.

**Comment by jziggy101** [comment]
I always wait for the app to no longer say its downloading feeds.   I agree if I only delete a few line at a time and then close the app when I open it again those changes are saved.  But sometime no matter how slow I do thing in the app it suddenly just closes.  I have tried running it as an Administrator but that did not make any difference.

**Comment by Rex000** [comment]
It crashes even in the background.

**Comment by jziggy101** [comment]
I ran the Windows 10 compatibility check on the QuiteRSS app and it says this app is a Windows 8 application.  So I told Windows 10 to always run this app as a Windows 8 app.  So far no crashes.  I will post any results I get from this testing.

**Comment by vitusson** [comment]
> Maybe you don't have a lot of feeds. Either that or the app just lost it's compatibility with Windows.

Don't know what a lot is. I have 112. But from what I read here, it looks more like a problem on Windows machines.

**Comment by jziggy101** [comment]
i was looking thru the app menu options.  In the Tools there is an option called Cleanup.  I ran that option and there was a great deal of disk activity while the app was running.  Then in the Tools section I clicked on 'FEEDS' and I changed some options in there for lower activity and only loading the Feeds less often and setting the date where QuiteRss only loads new feeds after Feb 18 and checked the box "Avoid adding news before the date into the database".    I closed the app and once reopened I have not had a sudden closure in 2 days now.

**Comment by Rex000** [comment]
Tried all of that still crashes. Sick and tired of this crap and I'm just stuck with it.

**Comment by ricardonevesmusic** [comment]
> Tried all of that still crashes. Sick and tired of this crap and I'm just stuck with it.

If you're on Windows 10, you can try this:

>If it doesn't open:
>
>Go to QuiteRss.ini in AppData and switch the option "reopenFeedStartup" from "true" to "false".

I was trying everything (deleting cache, reading the "debug.log", changing "compatibility settings") and the app didn't get past the 90% mark.

So I then suddenly something popped into my head.

Basically I started asking questions, like a problem-solving type of thing.

I asked myself: "What else haven't I tried before?"

And then suddenly an answer came into my head:

"If this app was working fine/right before it started to crash on load/startup, then compatibility settings and all the other stuff I've tried before must not be the issue."

Then something else popped into my head:

"What if I go to "QuiteRss.ini" config file?"

Well ...

I did go there and changed the option "reopenFeedStartup" from "true" to "false".

It turns out that something (the last feed I've opened) must have been corrupted and was causing the software not to launch/start.

The option "reopenFeedStartup" translated to everyday language (or at least one everyone can understand), stands for "Open Last Feed on Startup/Launch" and that's basically what it does.

This fixed the issue for me.

Hope it does it for you.

**Comment by birdie-github** [comment]
An active fork with full Qt6 support is here: https://github.com/birdie-github/notquiterss

You're welcome to try it.
